table_version-loader.bash

```
@@ -1,54 +1,68 @@
 #!/usr/bin/env bash
 
-TGT_SCHEMA=table_version
-TGT_DB=
-EXT_MODE=on
-EXT_NAME=table_version
-EXT_DIR=@@LOCAL_SHAREDIR@@
-TPL_FILE=
-VER=
+set -o errexit -o noclobber -o nounset -o pipefail
+shopt -s failglob inherit_errexit
 
-if test -n "$TABLE_VERSION_EXT_DIR"; then
-  EXT_DIR="$TABLE_VERSION_EXT_DIR"
+tgt_schema=table_version
+tgt_db=
+ext_mode=on
+ext_name=table_version
+ext_dir=@@LOCAL_SHAREDIR@@
+tpl_file=
+ver=
+
+if test -n "$TABLE_VERSION_EXT_DIR"
```

This will fail if `TABLE_VERSION_EXT_DIR` is *unset* rather than just empty.